File: 프로그래머스/Level1/같은 숫자는 싫어.py
# https://programmers.co.kr/learn/courses/30/lessons/12906

def solution(arr):
    temp = arr[0]
    answer = [temp]
    for i in range(1, len(arr)):
        if temp == arr[i]:
            continue
        temp = arr[i]
        answer.append(temp)
    return answer

File: 프로그래머스/Level1/같은 숫자는 싫어.js
// https://programmers.co.kr/learn/courses/30/lessons/12906

const solution = (arr) => {
	let answer = [arr[0]];
	let temp = arr[0];
	for (let i = 1; i < arr.length; i++) {
		if (arr[i] !== temp) {
			answer.push(arr[i]);
			temp = arr[i];
		}
	}
	return answer;
};
